webapp/channels/src/components/admin_console/team_channel_settings/team/details/team_profile.tsx

```diff
@@ -148,17 +147,23 @@ export function TeamProfile({team, isArchived, onToggleArchive, isDisabled, save
                         </div>
                         <div className='team-desc-col'>
                             <div className='row row-bottom-padding'>
-                                <FormattedMarkdownMessage
-                                    id='admin.team_settings.team_detail.teamName'
-                                    defaultMessage='**Team Name**:'
+                                <FormattedMessage
+                                    id='admin.teamSettings.teamDetail.teamName'
+                                    defaultMessage='<b>Team Name</b>:'
+                                    values={{
+                                        b: (chunks: string) => <b>{chunks}</b>,
```

Comment:
@M-ZubairAhmed In the example you used `<strong>` and here we are using `<b>`. Should we follow any particular standard on this? or is it a case by case basis?

Reply:
Oh lets open a thread in dev channel for this